### PR TITLE
Automatically close toastr box on timeout if animation is failed

### PR DESCRIPTION
**Pull request by jalkoby**
I've made the second implementation - based on timeout, so a user is free to used own transitions. Please look at it(info toastr in the demo app) as I'm starting to like it more than the current

### CODE REVIEW

**Comment by diegoddox** [comment]
What will happen if the the `toastr` has an option `timeOut:0`? I guess that it will be removed anyway after the [timeout](https://github.com/diegoddox/react-redux-toastr/pull/117/commits/e7e46bdf1c40e56f596a53f371cb285b4cec9d7b#diff-2b4ca49d4bb0a774c4d4c1672d7aa781R83) is called.

**Comment by jalkoby** [comment]
if set `timeOut: 0` a toast box stays opens forever as on end of transition never called 
 `if(0) === if(false)`

**Comment by diegoddox** [comment]
True that!, `on end of transition never called`

File: development/Menu.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ export default class Menu extends React.Component {
               position: 'bottom-right',
               timeOut: 10000,
               transitionIn: 'bounceInDown',
-              transitionOut: 'bounceOutUp'
+              transitionOut: 'bounceOutU'
```
Comment (XanderDwyl) [inline development/Menu.js:52]: wrong spelling, might be `bounceOutUp`?

**Review by XanderDwyl — CHANGES_REQUESTED** [review]

File: development/Menu.js (hunk 1 above)
Comment (diegoddox) [reply development/Menu.js:52]: I think that was by design in order to test the timeout function.

Comment (XanderDwyl) [reply development/Menu.js:52]: 👍

Comment (jalkoby) [reply development/Menu.js:52]: Exactly 😎

Comment (jalkoby) [reply development/Menu.js:52]: The idea behind this pr is to allow a user to use any css class, but close a popup in an invalid case as well

**Comment by VincentN** [comment]
Hey @diegoddox, will this be merged in? Without the typo of course. :D Or is this just a "show case" for a future enhancement?

**Comment by jalkoby** [comment]
It was a show case for the automatic close case

**Comment by diegoddox** [comment]
I'm will not be able to test but will merge!!

**Pull request merged**